### ISSUE TITLE
Auto complete and 'using namespace' inside a function

### ISSUE THREAD

**Issue by Muzza** [slynch8/10x#911]
See example, autocomplete does not handle 'using namespace' inside functions

```cpp
namespace NS1
{
	namespace NS2
	{
		void MyFunc( void )
		{
		}
	};
};

void Test1( void )
{
	using namespace NS1;
	using namespace NS2;
	// start to type, MyFunc does not appear in the autocomplete
}

using namespace NS1;
using namespace NS2;

void Test2( void )
{
	// start to type, MyFunc appears in the autocomplete
}
```

**Comment by Muzza** [comment]
I notice 'using namespace' in headers got added in the last update. Good time to support it in functions too?

**Comment by slynch8** [comment]
sure, turned out to be something simple.
fixed (0.32.95)
